lib/open_food_network/i18n_config.rb

```diff
@@ -5,6 +5,13 @@ module OpenFoodNetwork
   # Currently, language settings are read from the environment.
   # See: config/application.yml
   class I18nConfig
+    # Users don't need to select the already selected locale.
+    def self.locale_options
+      locales = selectable_locales
+      locales.delete(I18n.locale.to_s)
+      locales
```

Comment:
Cool, that's great, too. Sorry I didn't realise about the return value of `delete`. Another way to write this:
```suggestion
      selectable_locales - [I18n.locale.to_s]
```

Reply:
Way much better: thanks a lot!